packages/google_mobile_maps/lib/google_mobile_maps.dart

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
+
+import 'platform_overlay.dart';
+
+final MethodChannel _channel =
+    const MethodChannel('plugins.flutter.io/google_mobile_maps')
+      ..invokeMethod('init');
+
+/// A GoogleMaps geographical location.
+class Location {
+  final double latitude;
+  final double longitude;
+
+  const Location(this.latitude, this.longitude);
+
+  dynamic _toJson() => <dynamic>[latitude, longitude];
+}
+
+/// A GoogleMaps zoom value.
+class Zoom {
```

Comment:
Is it worth it wrapping a single double value?

Reply:
I think it might be. The underlying API has a particular range of valid values and there are certain approximate levels of detail associated to some of the values (https://developers.google.com/maps/documentation/static-maps/intro#Zoomlevels).